file_store.go

```diff
@@ -16,82 +16,43 @@ limitations under the License.
 package credentials
 
 import (
-	"context"
-	"errors"
-	"fmt"
-	"strings"
-
-	"github.com/oras-project/oras-credentials-go/internal/config"
-	"oras.land/oras-go/v2/registry/remote/auth"
+	orascreds "oras.land/oras-go/v2/registry/remote/credentials"
 )
 
 // FileStore implements a credentials store using the docker configuration file
 // to keep the credentials in plain-text.
 //
 // Reference: https://docs.docker.com/engine/reference/commandline/cli/#docker-cli-configuration-file-configjson-properties
-type FileStore struct {
-	// DisablePut disables putting credentials in plaintext.
-	// If DisablePut is set to true, Put() will return ErrPlaintextPutDisabled.
-	DisablePut bool
-
-	config *config.Config
-}
+//
+// Deprecated: This type is now simply [credentials.FileStore] of oras-go.
+//
+// [credentials.FileStore]: https://pkg.go.dev/oras.land/oras-go/v2/registry/remote/credentials#FileStore
+type FileStore = orascreds.FileStore
```

Comment:
Can we just use `credentials` instead of `orascreds`? I think it will impact the godoc rendering.

Reply:
That works. Nice!